mutiny-wasm/src/lib.rs

```diff
@@ -87,6 +87,8 @@ impl MutinyWallet {
         user_esplora_url: Option<String>,
         user_rgs_url: Option<String>,
         lsp_url: Option<String>,
+        lsp_connection_string: Option<String>,
+        lsp_token: Option<String>,
```

Comment:
well you can check the mutiny-web `lsps` branch to see how I did it for now.

right now the logic is:

if `lsp_url` is set then it's `voltage` flow.
if `lsp_connection_string` is set then it's `lsps` flow.

But yeah, it's _kind_ of awkward this way. There's no spec here so we're kind of making it up as we go.  I think as for how it should look in an ideal state for an advanced user who wants to switch LSPs is to support a dropdown that has mutiny pre-configured lsps (voltage, c=, whoever) and the last option is "Custom LSPS LSP". When that last option is selected it exposes two textboxes for providing connection string and token.

You could come up with some url scheme to combine them into one but it's not some standard. Could have pubkey:token@ip:port as a standard way to combine it, using the token almost like a basic auth password and pubkey as the username.

I'm not familiar with what the voltage style LSP_URL is but I imagine in theory you could combine these schemes. I don't think a user will ever need to enter a manual alternative voltage flow LSP though since that won't really exist.

We can talk about this on the call later today